### PR TITLE
don't export embedded textures with original, remove triangle warning

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3397/HighFidelity-Beta-PR11393-608f8196c62cde260d052d7ba7f3a33178538740.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3397/HighFidelity-Beta-PR11393-608f8196c62cde260d052d7ba7f3a33178538740.exe

File: libraries/baking/src/FBXBaker.cpp [hunk 1]
```diff
@@ -244,7 +244,7 @@ QString texturePathRelativeToFBX(QUrl fbxURL, QUrl textureURL) {
 QString FBXBaker::createBakedTextureFileName(const QFileInfo& textureFileInfo) {
     // first make sure we have a unique base name for this texture
     // in case another texture referenced by this model has the same base name
-    auto nameMatches = _textureNameMatchCount[textureFileInfo.baseName()];
+    auto& nameMatches = _textureNameMatchCount[textureFileInfo.baseName()];
```
Comment (Atlante45) [inline libraries/baking/src/FBXBaker.cpp]: Is it possible for another thread to insert into that hash?

Comment (birarda) [reply libraries/baking/src/FBXBaker.cpp]: Ah, it wasn't before thanks to the FBX SDK but it is now. I'll mutex it.

Comment (birarda) [reply libraries/baking/src/FBXBaker.cpp]: The same FBXBaker will always call `createBakedTextureFilename` from the same thread.

**Pull request merged**